### ISSUE TITLE
Was banned on the discord for no reason.

### ISSUE THREAD

**Issue by OrangeSodaPop** [ObliqueNET/Server#1068]
My discord ign is OrangeSoda#1769 for reference. I was banned on the discord for no reason, the only person (Admiral Crunch) who contacted me about it didn’t know why I was banned either. He first assumed it was because I posted a discord crashing video except for the fact I only mentioned I have another version of that video that so happens to crash discord. I didn’t post that however and I don’t know what else could of got me banned. if there is a reason why then it should of been mentioned to me either via dm or other such method. Thanks for your time and I hope this misunderstanding gets cleared up.

-Orange

**Comment by Rasgnarok** [comment]
The gif you posted, a short link video, actually did crash me. Only after I deleted your message did it stop auto-crashing my Discord app. I am not sure why you'd even post that in our most trafficked channel, #public. You can DM me about this if you want, but I did two separate bans through my phone, and yours canceled the reminder of my crash loop.

**Comment by OrangeSodaPop** [comment]
A slight problem with banning me for this one, I didn’t post a gif. Like you said, I posted a short link video. A MOV to be specific, which is a screen recording. How this crashed your discord is beyond me but my whole reason for posting it was someone posting an emote or possibly a small gif. I checked my video and it didn’t crash me on my phone or my computer. However if my video somehow crashed you, I apologize because I didn’t know that it could physically crash somebody. I linked what I posted below. 

https://user-images.githubusercontent.com/57567713/114061988-018bf380-9865-11eb-9dbc-236b6bc257a2.mov

-Orange
